test/extended/dns/dns.go

```diff
@@ -492,7 +498,7 @@ var _ = Describe("[sig-network-edge] DNS", func() {
 
 		// Run a pod which probes DNS and exposes the results by HTTP.
 		By("creating a pod to probe DNS")
-		pod := createDNSPod(f.Namespace.Name, cmd)
+		pod := createDNSPod(oc, f.Namespace.Name, cmd)
```

Comment:
If you follow https://github.com/openshift/origin/pull/29231/files#r1817010267, then you can:
```suggestion
		pod := createDNSPod(f.Namespace.Name, cmd, nodeselector)
```